### DISCUSSION TITLE
rsbuild build - minimum info

### DISCUSSION THREAD

**Discussion by iamravisingh** [web-infra-dev/rsbuild#2838] [Q&A] [answered]
When running `rsbuild build`. I want to show minimum info, say I want to skip file listing ( dist folder )or I want to add more info on time and size. 
Is that possible? I can't find anything related to this in docs

**Comment by chenjiahan** [comment, marked as answer]
- Skip file listing: https://rsbuild.dev/config/performance/print-file-size
- For adding more info, you can use Rsbuild plugin: https://rsbuild.dev/plugins/dev/index

**Reply by iamravisingh** [reply]
@chenjiahan Thanks for the quick response, this helps
